test/date.js

```diff
@@ -17,7 +17,13 @@ describe('date', () => {
     should.not.exist(util.date('abc', 'America/New_York'))
   })
   it('should return a valid date from string', async () => {
```

Comment:
Can you add tests that:
- ensure dates with an existing timezone are either converted, or not modified
- test ISO string inputs

Should also probably include changes in docs since this behavior is a little complex to describe